### PR TITLE
Fix AI being able to use ticket machine

### PR DESCRIPTION
**Pull request by Nazsgull**
## What Does This PR Do
Fixes the AI being able to interact with the recently added ticket machine.
Other commits in the admin.txt is me messing up and fixing it.

## Why It's Good For The Game
If the AI interacted with the ticket machine, a ticket would magically appear at its core. With this PR it instead cannot interact with it, fixing the bug. About the admin.txt thingy... I'm no admin, so I better not appear there.

## Changelog
:cl:
remove: AI cannot interact with ticket machine anymore.
/:cl:

### CODE REVIEW

**Comment by AffectedArc07** [comment]
The checks failing is not your fault. GitHub is currently hitting the shitter.

File: config/example/admins.txt [hunk 1]
```diff
@@ -5,4 +5,4 @@
 # Ranks can be anything defined in admin_ranks.txt           	     #
 ######################################################################
 
-# example - Hosting Provider
+# example - Hosting Provider
```
Comment (farie82) [inline config/example/admins.txt:8]: Do please revert this change. Itll mess up the history etc otherwise.
You can change it using notepad or such and just add a new line if vscode is being a shit.
Or you can setup vscode to add new lines at the end of files

File: code/modules/paperwork/ticketmachine.dm [hunk 2]
```diff
@@ -167,9 +167,13 @@
...
 		user.IgniteMob()
 
+
+/obj/machinery/ticket_machine/attack_ai(mob/user)//If the user is the AI
+	return //do nothing
+
 /obj/item/ticket_machine_ticket
 	name = "Ticket"
-	desc = "A ticket which shows your place in the Head of Personnel's line. Made from Nanotrasen patented NanoPaper�. Though solid, its form seems to shimmer slightly. Feels (and burns) just like the real thing."
+	desc = "A ticket which shows your place in the Head of Personnel's line. Made from Nanotrasen patented NanoPaper�. Though solid, its form seems to shimmer slightly. Feels (and burns) just like the real thing."
```
Comment (farie82) [inline code/modules/paperwork/ticketmachine.dm:176]: Also revert this change please. Looks like you are using a different character set and thus saving it corrupted the character there

**Comment by KasparoVy** [comment]
AI wants job change to clown
